modules/virt-single-node-cluster.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * virt/install/preparing-cluster-for-virt.adoc
+
+[id="virt-single-node-cluster_{context}"]
+= Changes in {VirtProductName} behavior on a single node cluster
+
+By default, you can use {VirtProductName} with single node clusters, also known as xref:../../operators/operator_sdk/osdk-ha-sno.adoc#osdk-ha-sno[Single Node OpenShift (SNO)]. However, these clusters are not configured for high-availability operation, which results in significant changes to the behavior of the cluster:
+
+* {VirtProductName} components have only one replica, and `PodDisruptionBudgets` are skipped.
+* Live migration is not available. There is no supported way to migrate a virtual machine to the same node that it is already on.
```

Comment:
Per minimalism principles, I think saying "Live migration is not supported" should be enough here.